### PR TITLE
Added logInfo with exception

### PR DESCRIPTION
**Pull request by ludwikkazmierczak**
- Added logInfo with exception
- Reformatted code

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/common/core/helpers/ArticlePurger.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package com.wikia.webdriver.common.core.helpers;
 
+import com.wikia.webdriver.common.logging.PageObjectLogging;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/core/helpers/ArticlePurger.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.logging.PageObjectLogging'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by nikodamn** [comment]
LGTM

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Pull request merged**